src/graph/LookupExecutor.cpp

```diff
@@ -316,10 +316,7 @@ LookupExecutor::findValidIndex() {
 void LookupExecutor::stepEdgeOut() {
     auto *sc = ectx()->getStorageClient();
     auto filter = Expression::encode(sentence_->whereClause()->filter());
```

Comment:
Could we unify the `stepEdgeOut` and `stepVertexOut` to `stepOut` ?

Reply:
perhaps `stepOut` is not very suitable for this context, ·lookUp· is a good name? 

What do you think?

Reply:
> perhaps `stepOut` is not very suitable for this context, ·lookUp· is a good name?
> 
> What do you think?

Aha, I think the setpOut is good . because it unify with go statement.